src/iexamine.cpp

```diff
@@ -1172,7 +1172,7 @@ void iexamine::flower_poppy(player *p, map *m, int examx, int examy)
     m->spawn_item(examx, examy, "poppy_bud");
 }
 
-void iexamine::flower_blubell(player *p, map *m, int examx, int examy)
+void iexamine::flower_bluebell(player *p, map *m, int examx, int examy)
```

Comment:
Just ran a check and it looks OK on a read: the name asks for an e, the handler translates that to non-e.  *shrug*  Will test in the mergetest and see.